packages/h5wasm/src/h5wasm-api.ts

```diff
@@ -60,14 +67,29 @@ export class H5WasmApi extends ProviderApi {
     return h5wDataset.value;
   }
 
+  public async getValue(params: ValuesStoreParams): Promise<unknown> {
+    const value = await this.fetchValue(params);
+
+    const { dataset } = params;
+    if (!hasBoolType(dataset)) {
+      return value;
+    }
+
+    return convertToBool(value);
+  }
```

Comment:
This is the issue I mentioned in #1111: boolean arrays get returned as `Int8Array`. 

We therefore need to convert both dataset and attribute boolean values. Perhaps this should be done in `h5wasm` instead ?

Reply:
I can't find the reference to this problem in #1111, but it seems to me that this should indeed be fixed in `h5wasm`.

Reply:
I was talking about my comment:

> I still have some issues with boolean which is why it is not included in this PR